notebooks/curation_api/python/src/collection.py

```diff
@@ -82,3 +82,18 @@ def get_collections(visibility: str = "PUBLIC") -> list:
     res = requests.get(url, headers=headers, params={"visibility": visibility})
     res.raise_for_status()
     return res.json().get("collections")
+
+
+def update_collection(collection_uuid: str, collection_form_metadata: dict) -> None:
+
+    url = url_builder(f"/collections/{collection_uuid}")
+    headers = get_headers()
+    try:
+        res = requests.put(url, headers=headers, data=json.dumps(collection_form_metadata))
```

Comment:
probably already on your radar, but just flagging that this is a `patch` request now with Trent's latest merged PR!